### DISCUSSION TITLE
Error fetching data

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#32] [Q&A] [answered]
HI
I made some changes with the cool picture edit tool. After saving I got this error:
![grafik](https://github.com/user-attachments/assets/21225b86-3bf7-48a7-a27d-884ad3056b85)

Now the system stocks after click on "Allgemeine Daten + Dateien". The counter of files is 0 instead of 3.
After delete cache I can work till I click again on it.

What could It be?

Thanks 
Gregor

**Comment by marcantondahmen** [comment]
Hi, it looks like a network issue or something else blocking communication to the server. What does the network tab in the browser dev tools say?

**Comment by cracksilver** [comment]
Did you mean this one ?
![grafik](https://github.com/user-attachments/assets/70342416-cae5-4485-91bd-3a3e4c26189d)

**Comment by marcantondahmen** [comment]
Yes, that one. Looks like the service is unavailable when requesting the files endpoint. I guess that your server is running out of memory and crashes the process when generating thumbnails.

This can have two reasons. Either the image you're saving is very large or your server has a memory limit set for PHP that is too low. 

In both cases I would recommend to increase the memory limit. Either in the `.htaccess` or `nginx.conf`, depending on the kind of server, Apache or Nginx, you're using.

**Reply by cracksilver** [reply]
Okey. It's good to know it isn't a problem with automad itself.
The 3 files I uploades and edited were very big, that's true. 

Since I do actually not have root access on this server I can't change something at the nginx.conf. I have a look what I can do in the .htaccess.

By the way, the memory limit seems not that bad:
![grafik](https://github.com/user-attachments/assets/eba44140-eadc-49f4-a356-922c08c29b88)

**Comment by cracksilver** [comment]
ahh... do you think I should uncomment this 4 lines?
![grafik](https://github.com/user-attachments/assets/b3fd05d8-4de7-40c6-a099-fc575a3f1631)

**Comment by marcantondahmen** [comment]
This are only the upload limits. You need to have more memory to manipulate and resize images. And uncommenting these lines only has an effect on Apache servers. The configuration file depends on the server you use.

I can only recommend using images with an appropriate resolution. This will probably solve already the memory issue.

**Reply by cracksilver** [reply, marked as answer]
> I can only recommend using images with an appropriate resolution. This will probably solve already the memory issue.

yes I think also that's the best way. 
thx
